presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+      parameters:
+      - description: The request body
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/ScannerCronJobRequest'
+      - format: int64
+        in: path
+        name: projectId
+        required: true
+        type: integer
+        x-go-name: ProjectId
+      responses:
+        "200":
+          $ref: '#/responses/scheduleScannerCronjobResponse'
+      tags:
+      - schedule-scanner-cronjob
+  /projects/{projectId}/schedule-streams-job:
+    post:
+      description: Schedule streams job
+      operationId: scheduleStreamsJobRequest
+      parameters:
+      - in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/StreamsJobRequest'
+      - format: int64
+        in: path
+        name: projectId
+        required: true
+        type: integer
+        x-go-name: ProjectId
+      responses:
+        "200":
+          $ref: '#/responses/scheduleStreamsJobResponse'
+      tags:
+      - schedule-streams-job
+  /templates/{projectId}/analyze/{templateId}:
+    delete:
+      description: Delete analyze template
+      operationId: deleteAnalyzeTemplateRequest
+      parameters:
+      - in: path
+        name: projectId
+        required: true
+        type: string
+        x-go-name: ProjectId
+      - in: path
+        name: templateId
+        required: true
+        type: string
+        x-go-name: TemplateId
+      responses:
+        "200":
+          $ref: '#/responses/templateGeneralResponse'
+        "400":
+          $ref: '#/responses/templateGeneralError'
+      tags:
+      - template
+      - analyze
+    get:
+      description: Get analyze template
+      operationId: getAnalyzeTemplateRequest
+      parameters:
+      - in: path
+        name: projectId
+        required: true
+        type: string
+        x-go-name: ProjectId
+      - in: path
+        name: templateId
+        required: true
+        type: string
+        x-go-name: TemplateId
+      responses:
+        "200":
+          $ref: '#/responses/templateGeneralResponse'
+        "400":
+          $ref: '#/responses/templateGeneralError'
+      tags:
+      - template
+      - analyze
+    post:
+      description: Create analyze template
+      operationId: postAnalyzeTemplateRequest
+      parameters:
+      - description: The request body
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/AnalyzeTemplate'
+      - in: path
+        name: projectId
+        required: true
+        type: string
+        x-go-name: ProjectId
+      - in: path
+        name: templateId
+        required: true
+        type: string
+        x-go-name: TemplateId
+      responses:
+        "200":
+          $ref: '#/responses/templateGeneralResponse'
+        "400":
+          $ref: '#/responses/templateGeneralError'
+      tags:
+      - template
+      - analyze
+    put:
+      description: Update analyze template
+      operationId: putAnalyzeTemplateRequest
```

Comment:
Can we call these "createOrUpdate*****Template" instead of "put"?

Reply:
PUT operations will only update the templates, and will receive error if the id does not exist. So renamed to update***Template